tests/resources/eloads/ELOAD_55/original_config.yml

```diff
@@ -4,31 +4,39 @@ brokering:
       '107': SAMEA8868451
     date: &id001 2021-05-04 11:48:57.230908
     pass: true
+  analyses:
```

Comment:
If this is being used to test the config upgrade, it should still be the old-style config right?